c2corg_ui/static/partials/cards/images.html

```diff
@@ -0,0 +1,16 @@
+<a ng-href="{{cardCtrl.createURL()}}" ng-init="doc = cardCtrl.doc; locale = cardCtrl.locale;">
+  <img ng-src="{{cardCtrl.createImg()}}" alt="{{locale['title']}}">
```

Comment:
> SM was too small and distorted

I guess you mean SI.
Isn't MI a bit too big for cards? Why SI images are distorted? You mean that's because they are squared (are they?)? What's the problem exactly?